[2025-02-04T23:04:40.692+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-04T23:04:40.703+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_dbt-model_credit_fraud.model_credit_fraud.agg_locations_risk_score_run manual__2025-02-04T22:52:25.583411+00:00 [queued]>
[2025-02-04T23:04:40.710+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_dbt-model_credit_fraud.model_credit_fraud.agg_locations_risk_score_run manual__2025-02-04T22:52:25.583411+00:00 [queued]>
[2025-02-04T23:04:40.711+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-02-04T23:04:40.719+0000] {taskinstance.py:2330} INFO - Executing <Task(DbtRunLocalOperator): model_credit_fraud.agg_locations_risk_score_run> on 2025-02-04 22:52:25.583411+00:00
[2025-02-04T23:04:40.723+0000] {standard_task_runner.py:63} INFO - Started process 910 to run task
[2025-02-04T23:04:40.725+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_dbt-model_credit_fraud', 'model_credit_fraud.agg_locations_risk_score_run', 'manual__2025-02-04T22:52:25.583411+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/dag_dbt-model_fraud_credit.py', '--cfg-path', '/tmp/tmpitpy5puy']
[2025-02-04T23:04:40.726+0000] {standard_task_runner.py:91} INFO - Job 9: Subtask model_credit_fraud.agg_locations_risk_score_run
[2025-02-04T23:04:40.770+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_dbt-model_credit_fraud.model_credit_fraud.agg_locations_risk_score_run manual__2025-02-04T22:52:25.583411+00:00 [running]> on host eb20c43aeeb4
[2025-02-04T23:04:40.828+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_dbt-model_credit_fraud' AIRFLOW_CTX_TASK_ID='model_credit_fraud.agg_locations_risk_score_run' AIRFLOW_CTX_EXECUTION_DATE='2025-02-04T22:52:25.583411+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-04T22:52:25.583411+00:00'
[2025-02-04T23:04:40.830+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-04T23:04:42.405+0000] {local.py:220} INFO - dbtRunner is available. Using dbtRunner for invoking dbt.
[2025-02-04T23:04:42.406+0000] {local.py:436} INFO - Cloning project to writable temp directory /tmp/tmpl9r77aj5 from /usr/local/airflow/dags/dbt/localiza_dw
[2025-02-04T23:04:42.410+0000] {local.py:447} INFO - Partial parse is enabled and the latest partial parse file is /tmp/cosmos/dag_dbt-model_credit_fraud__model_credit_fraud/target/partial_parse.msgpack
[2025-02-04T23:04:42.414+0000] {config.py:348} INFO - Profile caching is enable.
[2025-02-04T23:04:42.420+0000] {base.py:84} INFO - Using connection ID '***' for task execution.
[2025-02-04T23:04:42.422+0000] {config.py:328} INFO - Profile found in cache using profile: /tmp/cosmos/profile/bdc35343eb23031ee554202b16407e794194afcc614f7ea62bc658c63495a211/profiles.yml.
[2025-02-04T23:04:42.422+0000] {local.py:408} INFO - Trying to run dbtRunner with:
 ['run', '--models', 'agg_locations_risk_score', '--project-dir', '/tmp/tmpl9r77aj5', '--profiles-dir', '/tmp/cosmos/profile/bdc35343eb23031ee554202b16407e794194afcc614f7ea62bc658c63495a211', '--profile', '***', '--target', 'prod']
 in /tmp/tmpl9r77aj5
[2025-02-04T23:04:42.434+0000] {logging_mixin.py:188} INFO - 23:04:42  Running with dbt=1.6.1
[2025-02-04T23:04:42.507+0000] {logging_mixin.py:188} INFO - 23:04:42  Registered adapter: postgres=1.6.1
[2025-02-04T23:04:42.775+0000] {logging_mixin.py:188} INFO - 23:04:42  Found 3 models, 1 snapshot, 5 tests, 1 source, 0 exposures, 0 metrics, 349 macros, 0 groups, 0 semantic models
[2025-02-04T23:04:42.778+0000] {logging_mixin.py:188} INFO - 23:04:42
[2025-02-04T23:04:42.848+0000] {logging_mixin.py:188} INFO - 23:04:42  Concurrency: 1 threads (target='prod')
[2025-02-04T23:04:42.849+0000] {logging_mixin.py:188} INFO - 23:04:42
[2025-02-04T23:04:42.853+0000] {logging_mixin.py:188} INFO - 23:04:42  1 of 1 START sql table model dm_credit_operations.agg_locations_risk_score ..... [RUN]
[2025-02-04T23:04:44.104+0000] {logging_mixin.py:188} INFO - 23:04:44  1 of 1 OK created sql table model dm_credit_operations.agg_locations_risk_score  [SELECT 6 in 1.25s]
[2025-02-04T23:04:44.113+0000] {logging_mixin.py:188} INFO - 23:04:44
[2025-02-04T23:04:44.114+0000] {logging_mixin.py:188} INFO - 23:04:44  Finished running 1 table model in 0 hours 0 minutes and 1.33 seconds (1.33s).
[2025-02-04T23:04:44.120+0000] {logging_mixin.py:188} INFO - 23:04:44
[2025-02-04T23:04:44.121+0000] {logging_mixin.py:188} INFO - 23:04:44  Completed successfully
[2025-02-04T23:04:44.122+0000] {logging_mixin.py:188} INFO - 23:04:44
[2025-02-04T23:04:44.123+0000] {logging_mixin.py:188} INFO - 23:04:44  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-02-04T23:04:44.862+0000] {local.py:138} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v10.json is above dbt-ol supported version 7. This might cause errors.
[2025-02-04T23:04:44.903+0000] {local.py:494} INFO - Inlets: [Dataset(uri='postgres://postgres-db:5432/localiza_dw.dm_credit_operations.stg_credit_fraud', extra=None)]
[2025-02-04T23:04:44.904+0000] {local.py:495} INFO - Outlets: [Dataset(uri='postgres://postgres-db:5432/localiza_dw.dm_credit_operations.agg_locations_risk_score', extra=None)]
[2025-02-04T23:04:44.905+0000] {local.py:595} INFO - Assigning inlets/outlets without DatasetAlias
[2025-02-04T23:04:44.905+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2025-02-04T23:04:44.917+0000] {dag.py:3954} INFO - Setting next_dagrun for dag_dbt-model_credit_fraud to None, run_after=None
[2025-02-04T23:04:44.941+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-04T23:04:44.962+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_dbt-model_credit_fraud, task_id=model_credit_fraud.agg_locations_risk_score_run, run_id=manual__2025-02-04T22:52:25.583411+00:00, execution_date=20250204T225225, start_date=20250204T230440, end_date=20250204T230444
[2025-02-04T23:04:44.982+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-02-04T23:04:44.997+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-04T23:04:44.999+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
